CMakeLists.txt

```diff
@@ -272,8 +272,8 @@ endif()
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/CMakeUninstall.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/CMakeUninstall.cmake"
                IMMEDIATE @ONLY)
-add_custom_target(uninstall COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/CMakeUninstall.cmake)
-set_target_properties(uninstall PROPERTIES FOLDER ${CMAKE_TARGETS_ROOT_FOLDER})
+add_custom_target(uninstall_colmap COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/CMakeUninstall.cmake)
```

Comment:
Need to resolve conflicting target in poselib.

Reply:
@vlarsson Maybe we can call the poselib one "uninstall_poselib" for consistency?

Reply:
Sure. I setup a PR here https://github.com/PoseLib/PoseLib/pull/86 that should be merged to master soon.
@pablospe setup the CMake for poselib and I don't really know exactly how it works, so it's probably best if he has a look as well :)

Reply:
Interesting problem. I will see there is a different solution. 

I was thinking we could check in poselib if the `uninstall` is already defined:
```
# Check if the uninstall target already exists
if(NOT TARGET uninstall)
    # Define the uninstall target
    add_custom_target(uninstall
        COMMAND ...
    )
endif()
```
and it would be responsibly of the user (in this case, colmap, to uninstall poselib properly).

----

One question. How do you include `poselib` into `colmap`? FYI, Poselib supports being used as submodule (see [here](https://github.com/PoseLib/PoseLib#use-library-as-dependency-in-an-external-project)). But I think there is another alternative it is not included in poselib `README` yet but relevant for this case, which it is:
```
include(FetchContent)
FetchContent_Declare(
    poselib
    GIT_REPOSITORY <URL-of-PoseLib>
    GIT_TAG <desired-commit-or-tag>
    # Other options as necessary
)
FetchContent_MakeAvailable(poselib)
```

This is the default for google test (what [they recommend](https://google.github.io/googletest/quickstart-cmake.html#set-up-a-project) for external project after cmake 3.11). This will download the code during compilation time, avoiding the need to install it, which makes the uninstall irrelevant.